CHANGELOG.develop

```diff
@@ -3615,6 +3619,21 @@ files (thanks to Daniel Nicolai)
 - Prevent =tmux-command= at GUI mode (thanks to Isaac Zeng)
 - Fixed regression in tmux by setting keybindings using =use-package=
   (thanks to Aaron Jensen)
+**** Transmission
+- Key bindings (thanks to Arif Er):
+  - ~SPC m d~ is to remove and delete torrent at point.
```

Comment:
this for example is strictly worse than evil-collections'.

removal is a destructive operation and is better bounded to D (S-d)